src/site/content/en/fast/signed-exchanges/index.md

```diff
@@ -396,54 +406,25 @@ serve the SXG in response. For instructions on setting up the Web Packager
 server, see [How to set up signed exchanges using Web
 Packager](/signed-exchanges-webpackager).
 
-In production, `webpkgserver` should not use a public endpoint. Instead, the
-frontend web server should forward SXG requests to `webpkgserver`. These
-[recommendations](https://github.com/google/webpackager/blob/main/cmd/webpkgserver/README.md#running-behind-front-end-edge-server)
-contain more information on running `webpkgserver` behind a frontend edge
-server.
-
-### Other tooling
-
-This section lists tooling alternatives to Web Packager. In addition to these
-options, you can also choose to build your own SXG generator.
-
+### Libraries
 
-- `sxg-rs`
+These libraries could be used to build your own SXG generator:
 
-  [`sxg-rs`](https://github.com/google/sxg-rs) is a collection of tools for generating SXGs using various serverless platforms.
-  Cloudflare Workers and Fastly Compute@Edge, and a Rust library that could be
-  adapted to other serverless platforms.
+- [`sxg_rs`](https://github.com/google/sxg-rs/tree/main/sxg_rs) is a Rust library for
+  generating SXGs. It is the most featureful, and is used as the basis for the
```

Comment:
```suggestion
  generating SXGs. It is used as the basis for the
```

Reply:
Why? How would you suggest we soft-recommend this one? Note that the previous revision discouraged `gen-signedexchange` "Due to its limited feature set".